var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -361,6 +367,10 @@ def cmake_args(self):
                 [define("TPL_ENABLE_METIS", False), define("TPL_ENABLE_ParMETIS", False)]
             )
 
+        if "+pamgen" in self.spec:
```

Comment:
The `if "+pamgen" in self.spec:` should not be required anymore with this change.

Reply:
I misunderstood and this is fixed.